assemblies/server/src/main/resources-filtered/etc/custom.properties

```diff
@@ -159,7 +159,8 @@ org.osgi.framework.system.packages.extra= \
  org.apache.hbase.thirdparty.*, \
  org.apache.hbase.thirdparty.com.google.common.primitives.*, \
  com.google.common.collect;version="12.0.0", \
- org.pentaho.di.core.namedcluster.*;version="${big-data-plugin.version}"
+ org.pentaho.di.core.namedcluster.*;version="${big-data-plugin.version}", \
+ com.hitachivantara.security.web.*;version="${hv-security-web.version}"
```

Comment:
As per our conversation in slack, check if its possible to only export the API packages.